scripts/generate-sw.js

```diff
@@ -0,0 +1,8 @@
+var path = require('path');
+var swPrecache = require('sw-precache');
+var __dirname = 'themes/aframe/source';
+
+swPrecache.write(path.join('scripts','sw.js'), {
```

Comment:
The code should be generating a precache of `themes/aframe/source` to a ServiceWorker in `scripts`